jax/_src/numpy/ufuncs.py

```diff
@@ -587,19 +587,119 @@ def tan(x: ArrayLike, /) -> Array:
   """
   return lax.tan(*promote_args_inexact('tan', x))
 
-@implements(np.arcsin, module='numpy')
+
 @partial(jit, inline=True)
 def arcsin(x: ArrayLike, /) -> Array:
+  r"""Compute element-wise inverse of trigonometric sine of input.
```

Comment:
For all three functions, we should say something about the branch cut convention used for complex inputs. Maybe the easiest thing would be to say that they follow the branch cut convention used by the associated NumPy function.

Reply:
Can we include that a note?

Reply:
A note is added for all the function about the branch cut convention used for complex inputs.